adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -145,18 +167,19 @@ - (void)getInterstitial {
     [strongConnector adapter:self didFailAd:error];
     return;
   }
-  [[GADMAdapterVungleRouter sharedInstance] initWithAppId:appID delegate:self];
+  GADMAdapterVungleInterstitial *__weak weakSelf = self;
```

Comment:
This doesn't do anything. Just because this reference is weak, it's still pointing to the object. If the class it's passed to keeps this as strong, it will still be a strong reference.

Reply:
@ericleich Got it, I will remove this redundant implementation. Thanks a lot for your guidance.  ^_^

Reply:
@ericleich  Done, thanks for your instruction. 😊